loan-agreement/src/main/java/de/weinbrecht/luc/bpm/architecture/loan/agreement/adapter/in/process/SendCrossSellingRecommendation.java

```diff
@@ -20,18 +19,16 @@ public class SendCrossSellingRecommendation {
     private final RecommendationTrigger recommendationTrigger;
     private final LoanAgreementQuery loanAgreementQuery;
 
-    @ZeebeWorker(type = SEND_CROSS_SELLING_RECOMMENDATION_TASK, fetchVariables = LOAN_AGREEMENT_NUMBER)
-    public void handleJobFoo(final JobClient client, final ActivatedJob job) {
-        Long loanAgreementNumber = ((Number) job.getVariablesAsMap().get(LOAN_AGREEMENT_NUMBER)).longValue();
-        String businessKey = (String) job.getVariablesAsMap().get(BUSINESS_KEY);
-        LoanAgreement loanAgreement = loanAgreementQuery.loadByNumber(new LoanAgreementNumber(loanAgreementNumber));
+    @ZeebeWorker(
+            type = SEND_CROSS_SELLING_RECOMMENDATION_TASK,
+            fetchVariables = { LOAN_AGREEMENT_NUMBER, BUSINESS_KEY },
```

Comment:
Same here - all @ZeebeVariable's are automatically fetched - no need to specify them in the header again